Firmware/mmu2_protocol_logic.cpp

```diff
@@ -316,6 +316,7 @@ StepStatus ProtocolLogic::CommandStep() {
         SendFINDAQuery();
         return Processing;
     case ScopeState::FINDAReqSent:
+        findaPressed = rsp.paramValue;
```

Comment:
:man_facepalming: I forgot about this one ... moreover - there are 2 idential pieces of code (in CommandStep and in IdleStep). May be we could make them one function if it saves some bytes.